lib/Dialect/TTMetal/Transforms/Passes.cpp

```diff
@@ -790,18 +789,17 @@ class TTIRToTTMetalDispatchRewriter : public OpRewritePattern<ttir::GenericOp> {
       return failure();
     }
 
-    SmallVector<Attribute> threadTypes = {
-        rewriter.getAttr<ttkernel::ThreadTypeAttr>(
-            ttkernel::ThreadType::Tensix),
-    };
+    auto tensixAttr = rewriter.getAttr<ttkernel::TensixConfigAttr>(
+        ttkernel::MathFidelity::HiFi4, false, false, false);
+    SmallVector<Attribute> kernelConfigs = {tensixAttr};
```

Comment:
Not sure how compiler will drive these long term, but for this merge this seems okay to me, since these match the `ComputeConfig` struct default values from `third_party/tt-metal/src/tt-metal/tt_metal/impl/kernels/kernel_types.hpp`that would have been used before your changes.